kolibri/plugins/setup_wizard/assets/src/views/onboarding-forms/SettingUpKolibri.vue

```diff
@@ -190,7 +190,12 @@
             window.sessionStorage.setItem(welcomeDimissalKey, false);
 
             Lockr.rm('savedState'); // Clear out saved state machine
-            redirectBrowser();
+            if (this.isOnMyOwnSetup) {
+              redirectBrowser(urls['kolibri:kolibri.plugins.learn:learn']());
```

Comment:
This creates an explicit link between the two plugins - if setup wizard is enabled and the learn plugin is not, this will throw an error, as this URL will be undefined.

I would suggest making no edits here, and instead just rely on the more general redirect behaviour that I suggest above.

Changing the name of the 'DEVICE_WELCOME_MODAL_DISMISSED' to just 'WELCOME_MODAL_DISMISSED' could be used to make this more general, and be reused in different plugins (although, you can just leave it with the same name, as I don't think it is terribly confusing if conceived as 'welcome to the device' not 'welcome to the device plugin'.)

Reply:
[note: after discussion, this was resolved by #11019]